docs/README.md

```diff
@@ -0,0 +1,26 @@
+# kit-docs
+
+This is a Next.js application generated with
+[Create Fumadocs](https://github.com/fuma-nama/fumadocs).
+
+Run development server:
+
+```bash
+npm run dev
+# or
+pnpm dev
+# or
+yarn dev
```

Comment:
Maybe mention this `--ignore-workspaces` malarkey here, and limit this to pnpm?

Reply:
Good shout. I'll update the README.